app/src/androidTest/java/com/example/mindyourpet/PetDatabaseTest.kt

```diff
@@ -0,0 +1,102 @@
+package com.example.mindyourpet
+
+import androidx.room.Room
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import com.example.mindyourpet.database.Pet
+import com.example.mindyourpet.database.PetDatabase
+import com.example.mindyourpet.database.PetDatabaseDao
+import com.example.mindyourpet.database.Reminder
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.assertEquals
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.IOException
+import kotlin.jvm.Throws
+
+
+/**
+ * This is not meant to be a full set of tests. For simplicity, most of your samples do not
+ * include tests. However, when building the Room, it is helpful to make sure it works before
+ * adding the UI.
+ */
+
+@RunWith(AndroidJUnit4::class)
+class PetDatabaseTest {
+
+    private lateinit var petDao: PetDatabaseDao
+    private lateinit var db: PetDatabase
+
+    @Before
+    fun createDb() {
+        val context = InstrumentationRegistry.getInstrumentation().targetContext
+        // Using an in-memory database because the information stored here disappears when the
+        // process is killed.
+        db = Room.inMemoryDatabaseBuilder(context, PetDatabase::class.java)
+            // Allowing main thread queries, just for testing.
+            .allowMainThreadQueries()
+            .build()
+        petDao = db.PetDatabaseDao
+
+        initializeDatabase()
+    }
+    fun initializeDatabase(){
```

Comment:
There should be a blank line before every method definition.

Reply:
I commented on this before, not sure why my replies aren't showing up but I said yes, understood.